repo/packages/P/portworx-confluent-kafka/400/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "1.3.2-2.6.0-5.1.2"
+  ],
+  "downgradesTo": [],
```

Comment:
hi @tusharraut1994 , you should also specify a value for downgrades to.

Reply:
@realmbgl We don't support downgrade so removed that. Do you want me to add that?

Reply:
hi  @tusharraut1994 , is this true across all your frameworks or just this one.

Reply:
@realmbgl, All frameworks.